extensions/container-image/container-image-docker/runtime/pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <artifactId>quarkus-container-image-docker</artifactId>
     <name>Quarkus - Container - Image - Docker</name>
-    <description>Extension for building container images with Docker</description>
+    <description>Build container images with Docker</description>
```

Comment:
I don't get what this one does..

Reply:
@geoand because you know what it does 😂

Reply:
I don't get if it let me build container images from my quarkus app or if it will add the magic to have my quarkus application built with docker..

Reply:
Well I assume you would look at the guide, right?

Reply:
Maybe `Build container images of your application with Docker`?

Reply:
applied